notebooks/00_core.ipynb

```diff
@@ -43,20 +43,21 @@
     "    \"\"\"\n",
```

Comment:
<p>"dataset is a cleaned subset of the <code>housing_gmaps_data_raw.csv</code>  dataset."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>